pom.xml

```diff
@@ -162,7 +162,7 @@
   </dependencies>
 
   <build>
-    <defaultGoal>clean verify apache-rat:check clirr:check checkstyle:check spotbugs:check javadoc:javadoc</defaultGoal>
+    <defaultGoal>clean verify apache-rat:check clirr:check</defaultGoal>
```

Comment:
Was it because we had too many checkstyle, spotbugs, and javadoc errors, that it was not worth fixing everything in this PR?

Reply:
@kinow, I am creating another PR actually which has a maven modular design. just to minimize clutter I removed these things. apologies for being late though, bit busy on the personal front from the last few weeks.

Reply:
Not a problem. @ameyjadiye is it alright if we merge this one then? Or does your other branch includes the work of this branch too, and merging/rebasing will be complicated later?

Reply:
@kinow Yes, it is ok to merge this PR, basically this PR was to just stabilize existing code. My other branch already having this code and is much stable and modular. it won't be issue to merge in future, if it comes, I'll take care of that :-)

Reply:
Thanks @ameyjadiye ! Just wanted to confirm it wouldn't cause any any issues merging this one. Done! :tada: